### PR TITLE
Add muse template (discord music bot)

### PR DESCRIPTION
**Pull request by d8vjork**
https://github.com/codetheweb/muse

### CODE REVIEW

File: templates/muse/index.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      serviceName: input.appServiceName,
+      env: [
+        `DISCORD_TOKEN=${input.discordToken}`,
+        `YOUTUBE_API_KEY=${input.youtubeApiKey}`,
+        `SPOTIFY_CLIENT_ID=${input.spotifyClientId}`,
+        `SPOTIFY_CLIENT_SECRET=${input.spotifyClientSecret}`,
+      ].join("\n"),
+      source: {
+        type: "image",
+        image: "codetheweb/muse:2.1"
```
Comment (Ponkhy) [inline templates/muse/index.ts:20]:
```suggestion
        image: input.appServiceImage
```

File: templates/muse/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - label: Documentation
+    url: https://github.com/codetheweb/muse#running
+contributors:
+  - name: Rubén Robles
+    url: https://github.com/D8vjork
+schema:
+  type: object
+  required:
+    - projectName
+    - appServiceName
```
Comment (Ponkhy) [inline templates/muse/meta.yaml:19]:
```suggestion
    - appServiceName
    - appServiceImage
```

File: templates/muse/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    projectName:
+      type: string
+      title: Project Name
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: discordbot
+    appServiceImage:
+      type: string
+      title: App Service Image
```
Comment (Ponkhy) [inline templates/muse/meta.yaml:34]:
```suggestion
    appServiceImage:
      type: string
      title: App Service Image
      default: codetheweb/muse:2.2.2
```

**Review by Ponkhy — CHANGES_REQUESTED** [review]
Please also add a new env. `DATA_DIR=./data` and the according volume.

**Comment by d8vjork** [comment]
Thanks for the details @Ponkhy !

File: templates/muse/index.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+export function generate(input: Input): Output {
+  const services: Services = [];
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      env: [
+        `DISCORD_TOKEN=${input.discordToken}`,
```
Comment (Ponkhy) [inline templates/muse/index.ts:13]:
```suggestion
        `DATA_DIR=./data`,
        `DISCORD_TOKEN=${input.discordToken}`,
```

**Comment by deiucanta** [comment]
@d8vjork @Ponkhy is this good to be merged?

**Comment by d8vjork** [comment]
> @d8vjork @Ponkhy is this good to be merged?

I think so, go ahead

**Comment by deiucanta** [comment]
Awesome, thanks!

**Pull request merged**